CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
    * REMOVED: Unused overloads of `to_response` function [#3167](https://github.com/valhalla/valhalla/pull/3167)
 
 * **Bug Fix**
+   * FIXED: Cache for gzipped elevation tiles [#3120](https://github.com/valhalla/valhalla/pull/3120)
```

Comment:
this is in the wrong section, since we've release since you prd this. i will fix it manually after merge